lib/array.js

```diff
@@ -148,11 +148,11 @@ module.exports = (api) => {
         done(err);
       } else {
         count++;
-        if (count === len) done();
+        if (count === len) done(null);
       }
     };
 
-    if (len < 1) return done();
+    if (len < 1) return done(null);
```

Comment:
@DzyubSpirit so it's for type safety? Just because `null` is a reference as well as a `function` but `undefined` is a value? Maybe you can find something in the web from authors/researchers of V8 and Mozilla?